packages/jest-cli/src/cli/index.ts

```diff
@@ -120,22 +120,20 @@ const readResultsAndExit = (
   if (globalConfig.forceExit) {
     if (!globalConfig.detectOpenHandles) {
       console.error(
```

Comment:
console.warn then (it still prints to stderr)?

Reply:
Looks like it does print to `stderr`, docs say it's literally just an alias for `console.error`. I'll switch it.